### PR TITLE
added coiote tutorials

### PR DESCRIPTION
**Pull request by malw4**

### CODE REVIEW

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_simply_onboard_and_provision_IoT_devices.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# How to simply onboard and provision IoT devices?
+
+If you've struggled with getting your IoT devices securely connected to the cloud, you're not alone – it's a common challenge for many developers. But here's the good news: we've got a straightforward solution, and using the Coiote IoT Device Management Platform can make it even easier.
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/How_to_simply_onboard_and_provision_IoT_devices.md:3]: > But here's the good news

`here's` suggest singular, but `good news` is plural

Comment (JZimnol) [reply docs/Coiote_IoT_DM/Coiote_videos/How_to_simply_onboard_and_provision_IoT_devices.md:3]: Also, at the beginning you're talking about `struggling` and `challenges`, but then you're saying `we can make it even easier`. That `even` doesn't make any sense.

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_simply_onboard_and_provision_IoT_devices.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+To get started, grab a board with Anjay IoT SDK, Zephyr Anjay Client, or any LwM2M Client. You can find the list [here](https://iotdevzone.avsystem.com/docs/LwM2M_Client/Getting_started/).
+
+In our tutorial, we'll guide you through registering your device and ensuring it's securely provisioned.
+We'll cover authentication, confirmed registration, and show you how to check your device's status.
+
+Connect your IoT devices with Anjay IoT SDK - LwM2M Client:
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/How_to_simply_onboard_and_provision_IoT_devices.md:10]: I'm not sure if the `Anjay IoT SDK - LwM2M Client` is a proper name - for me it's either `Anjay LwM2M Client` or `Anjay IoT SDK`.

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_simply_onboard_and_provision_IoT_devices.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+* [Anjay Client repository](https://github.com/AVSystem/Anjay)
+* [Anjay Zephyr Client repository](https://github.com/AVSystem/Anjay-zephyr-client)
+
+Coiote IoT Device Management - free registration:
+Test it out yourself by registering for a free Developer account [{{ coiote_short_name }}]({{ coiote_site_link }}/) user account.
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/How_to_simply_onboard_and_provision_IoT_devices.md:16]: It's in one line and looks weird.

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_simply_onboard_and_provision_IoT_devices.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Jyn7RrNHFWc?si=j6FYiyKSC7eqK0LY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/How_to_simply_onboard_and_provision_IoT_devices.md:19]: Can you center the video on the page?

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# How to remotely configure IoT devices?
+
+With IoT devices enabled with LwM2M protocol, you can use a set of standardized remote operations to configure your IoT devices remotely.
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md:3]: Twice `IoT Devices`, also `remote operations` to configure `remotely` sound weird too.

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+To prepare your device with LwM2M connectivity, grab Anjay IoT SDK, Zephyr Anjay Client, or any LwM2M Client. Check out more handy links below.
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md:5]: Same as above.

Also `Check out more handy links below` but there is no link here, so definitely not `more handy links`.

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+In our tutorial, we'll be exploring remote configuration options such as Read, Write, Observe, and Execute operations.
+
+Connect your IoT devices with Anjay IoT SDK - LwM2M Client:
+
+* [Anjay Client repository](https://github.com/AVSystem/Anjay)
+* [Anjay Zephyr Client repository](https://github.com/AVSystem/Anjay-zephyr-client)
+
+Coiote IoT Device Management - free registration:
+Test it out yourself by registering for a free Developer account [{{ coiote_short_name }}]({{ coiote_site_link }}/) user account.
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md:15]: Same as above.

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/3_EKJR6IMLs?si=NanXkrX8Dj_VTEfS" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md:17]: Same as above.

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+With IoT devices enabled with LwM2M protocol, you can use a set of standardized remote operations to configure your IoT devices remotely.
+
+To prepare your device with LwM2M connectivity, grab Anjay IoT SDK, Zephyr Anjay Client, or any LwM2M Client. Check out more handy links below.
+
+In our tutorial, we'll be exploring remote configuration options such as Read, Write, Observe, and Execute operations.
+
+Connect your IoT devices with Anjay IoT SDK - LwM2M Client:
+
+* [Anjay Client repository](https://github.com/AVSystem/Anjay)
+* [Anjay Zephyr Client repository](https://github.com/AVSystem/Anjay-zephyr-client)
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md:12]: Why only those two links?

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_perform_secure_Over-the-Air_updates.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
+# How to perform secure Over-the-Air (OTA) updates? (nRF 9160)
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/How_to_perform_secure_Over-the-Air_updates.md:1]: There is no such thing as `nRF 9160`. It's either `nRF9160` (which means the chip itself) ot `nRF9160 DK` (which means the Development Kit with the nRF9160 chip).

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_perform_secure_Over-the-Air_updates.md [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+
+The LwM2M standard makes Over-the-Air (OTA) updates straightforward, reducing the complexity of implementation and management. What's crucial is that it enhances security through authentication and encryption.
+
+To demonstrate the potential of a single update, we use an IoT device with nRF 9160 DK integrated with Anjay IoT SDK—a setup that enables LwM2M Connectivity.
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/How_to_perform_secure_Over-the-Air_updates.md:5]:
```suggestion
To demonstrate the potential of a single update, we will use an IoT device with nRF9160 DK integrated with Anjay IoT SDK— a setup that enables LwM2M Connectivity.
```

Also, what does `an IoT device with nRF9160 DK`? The board acts as an IoT Device itself so it doesn't make any sense for me. But maybe I am wrong...

File: docs/Coiote_IoT_DM/Coiote_videos/What_collected_IoT_data_can_you_find_in_the_Coiote.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
+# What collected IoT data can you find in the Coiote?
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/What_collected_IoT_data_can_you_find_in_the_Coiote.md:1]: Sound weird...
```suggestion
# What IoT data is collected and stored in the Coiote?
```
or `What types of IoT data is collected and stored in the Coiote?`

File: docs/Coiote_IoT_DM/Coiote_videos/What_collected_IoT_data_can_you_find_in_the_Coiote.md [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+
+The Coiote IoT Device Management Platform enables you to gather diverse data types through the standardized data model offered by the LwM2M protocol.
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/What_collected_IoT_data_can_you_find_in_the_Coiote.md:3]: `gather data through data model`? Maybe it's correct, I don't know, but from my point of view, `gather data inside data model` or something.

File: docs/Coiote_IoT_DM/Coiote_videos/What_collected_IoT_data_can_you_find_in_the_Coiote.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+- Device Information
+- Connectivity Status
+- Firmware and Software Versions
+- Configuration Data
+- Sensor Data
+- Location Data
+- Security Information
+- Battery Status
+
+It equips you with everything needed to monitor device health and collect essential data for IoT applications or external platforms.
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/What_collected_IoT_data_can_you_find_in_the_Coiote.md:16]:
```suggestion
It equips you with everything needed to monitor device's health and collect essential data for IoT applications or external platforms.
```
But I'm not sure.

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_monitor_IoT_devices_and_diagnose_issues.md [hunk 15]
```diff
@@ -0,0 +1,13 @@
+# How to remotely monitor IoT devices and diagnose issues?
+
+With Coiote IoT Device Management Platform, you can effortlessly monitor and troubleshoot all your IoT devices from one central location. Our tutorial will walk you through the real-time monitoring of data for each device and demonstrate how to manage an entire fleet of IoT devices, resolving issues remotely.
```
Comment (JZimnol) [inline docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_monitor_IoT_devices_and_diagnose_issues.md:3]:
```suggestion
With Coiote IoT Device Management Platform, you can effortlessly monitor and troubleshoot all your IoT devices from one central location. Our tutorial will walk you through the real-time data monitoring for each device and will demonstrate how to manage an entire fleet of IoT devices by resolving issues remotely.
```

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_simply_onboard_and_provision_IoT_devices.md (hunk 4 above)
Comment (malw4) [reply docs/Coiote_IoT_DM/Coiote_videos/How_to_simply_onboard_and_provision_IoT_devices.md:19]: Honestly? I can't xd

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_perform_secure_Over-the-Air_updates.md (hunk 11 above)
Comment (malw4) [reply docs/Coiote_IoT_DM/Coiote_videos/How_to_perform_secure_Over-the-Air_updates.md:5]: it's only to highlight the device

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md (hunk 9 above)
Comment (malw4) [reply docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md:12]: well on YouTube we also added the Devzone link but for Devzone I think we can have these two links. Overall, we wanted to highlight this configuration which we have in the video.

File: docs/Coiote_IoT_DM/Coiote_videos/How_to_perform_secure_Over-the-Air_updates.md (hunk 10 above)
Comment (JZimnol) [reply docs/Coiote_IoT_DM/Coiote_videos/How_to_perform_secure_Over-the-Air_updates.md:1]: Still, not every occurrence has been changed.

File: docs/Coiote_IoT_DM/Coiote_videos/What_collected_IoT_data_can_you_find_in_the_Coiote.md (hunk 12 above)
Comment (JZimnol) [reply docs/Coiote_IoT_DM/Coiote_videos/What_collected_IoT_data_can_you_find_in_the_Coiote.md]: The filename should be changed too, I think.

**Comment by JZimnol** [comment]
There is still one pending comment regarding `nRF9160 DK`.

**Pull request merged**